AccessibilityInsightsForAndroidService/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,9 +1,6 @@
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
-#
-# Thu Jan 16 11:25:59 PST 2020
+#Mon Aug 24 10:57:42 EDT 2020
```

Comment:
I removed the copyright notice because it wasn't appropriate for us to have added it in the first place; we never made any modifications to this file since it was pulled down during gradle install.